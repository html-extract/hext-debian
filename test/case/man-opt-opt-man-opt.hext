<a>
  <?span class:1_opti_span>
    <?span class:2_opti_span>
      <span class:3_mand_span>
        <?span class:4_opti_span/>
      </span>
    </span>
  </span>
</a>
