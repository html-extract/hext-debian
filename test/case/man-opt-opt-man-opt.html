<a href="">
  <span class="-span1">
    <span class="--span1">
      <span class="---span1">
        <span class="----span1"></span>
      </span>
    </span>
    <span class="--span2 dupl">
      <span class="---span1 dupl">
        <b class="error">nope</b>
      </span>
    </span>
  </span>
</a>
