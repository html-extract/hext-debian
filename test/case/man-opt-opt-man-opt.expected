{"1_opti_span":"-span1","2_opti_span":["--span1","--span2 dupl"],"3_mand_span":["---span1","---span1 dupl"],"4_opti_span":"----span1"}
